src/storage-control/src/convert.rs

```diff
@@ -27,6 +27,9 @@
 
 use crate::google;
 use gaxi::prost::{ConvertError, FromProto, ToProto};
+// This would fail if storage admin introduces LROs. We are ok with it, because
+// we know the service will not grow.
```

Comment:
```suggestion
// This would fail if storage admin introduces LROs. This is unlikely as 
// one of the motivations for `StorageControl` was to separate the LROs.
// Worst case, we can skip the offending functions until we have fixed
// this code to support them.
```

Reply:
Done.